src/commands/moderation/reason.ts

```diff
@@ -29,30 +33,148 @@ export default class implements Command {
 			throw new Error(i18next.t('common.errors.no_mod_log_channel', { lng: locale }));
 		}
 
-		const originalCase = await getCase(interaction.guildId!, args.case);
-		if (!originalCase) {
-			throw new Error(i18next.t('command.mod.common.errors.no_case', { case: args.case, lng: locale }));
-		}
-
 		if (args.reason.length >= 500) {
 			throw new Error(i18next.t('command.mod.common.errors.max_length_reason', { lng: locale }));
 		}
 
-		const case_ = await updateCase({
-			caseId: originalCase.caseId,
-			guildId: interaction.guildId!,
-			reason: args.reason,
-		});
-		await upsertCaseLog(interaction.guildId!, interaction.user, case_);
+		const lower = Math.min(args.case, args.lastcase ?? args.case);
+		const upper = Math.max(args.case, args.lastcase ?? args.case);
```

Comment:
It might be worth doing some validation here:

* Are they both positive? (idk if it's possible to provide negatives via Discord, but maybe worth validating just in case)
* Is `upper` > `lower`?

Reply:
As for `upper` > `lower` I felt like scrolling through the logs you can start on either end, hence the min/max approach (if they're equal that might look a little funky "cases from x to x (1 case)" but I don't think that's unexpected or a hindrance)

> idk if it's possible to provide negatives via Discord

It is, currently rejects by not finding that case, but I suppose that one can indeed cause an early return to not have to hit the DB.